docs/FrontierDevelopments-CAPI-endpoints.md

```diff
@@ -14,138 +16,373 @@ It's safest to store the token_type that comes back from the request for
-	1. `locName`: "SRV Scarab"
-	1. `loadout`: "starter"
-	1. `loadoutName`: "Starter" (Localised?)
+    1. `rebuilds`: How many spares???
+    1. `name`: 'testbuggy' for an SRV
+    1. `locName`: "SRV Scarab"
+    1. `loadout`: "starter"
+    1. `loadoutName`: "Starter" (Localised?)
 1. Other Ships data
+
+&nbsp;
+
+---
+
 ## Shipyard
+
+    	GET <https://companion.orerve.net/shipyard>
+
+Fetches the last visited shipyard for the authenticated commander
+
+1. `id` - Station ID
+1. `name` - Station name
+1. `outpostType` - What type of outpost it is, examples: `starport`, `fleetcarrier`
+1. `imported` - What commodities that are imported from this station (dictionary, commodity id and key)
+   - Example `"128049162": "Cobalt"`
+1. `exported` - What commodities that are exported from this station (dictionary, commodity id and key)
+   - Example `"128049162": "Cobalt"`
+1. `services` - What services the station provides and their status
+	1. `dock` - Status of Dock (`ok`, `unavailable`, `private`)
+	1. `contacts` - Status of Contacts (`ok`, `unavailable`, `private`)
+	1. `exploration` - Status of Exploration (`ok`, `unavailable`, `private`)
+	1. `commodities` - Status of Commodities (`ok`, `unavailable`, `private`)
+	1. `refuel` - Status of Refuel (`ok`, `unavailable`, `private`)
+	1. `repair` - Status of Repair (`ok`, `unavailable`, `private`)
+	1. `rearm` - Status of Rearm (`ok`, `unavailable`, `private`)
+	1. `outfitting` - Status of Outfitting (`ok`, `unavailable`, `private`)
+	1. `shipyard` - Status of Shipyard (`ok`, `unavailable`, `private`)
+	1. `crewlounge` - Status of Crewlounge (`ok`, `unavailable`, `private`)
+	1. `searchrescue` - Status of Search & Rescue (`ok`, `unavailable`, `private`)
+	1. `techbroker` - Status of Tech broker (`ok`, `unavailable`, `private`)
+	1. `stationmenu` - Status of Station menu (`ok`, `unavailable`, `private`)
+	1. `shop` - Status of shop (`ok`, `unavailable`, `private`)
+	1. `engineer` - Status of engineers (`ok`, `unavailable`, `private`)
+1. `economies` - What different types of economies you can expect from the station (dictionary, economyId as key)
+	- Example `"23": { "name" : "HighTech", "proportion": 0.7 }`
+1. `prohibited` - What commidities that are prohibited on this station (dictionary, commodityId and key)
+   - Example `"128049162": "Cobalt"`
+1. `modules` - The list of modules on this station (dictionary, moduleId as key)
+	1. `id` - ModuleId
+	1. `category` - What category this module is
+	1. `name` - Name for the module
+	1. `cost` - Module cost
+	1. `sku` - The SKU for the module
+	1. `stock` - How many are available of this item, `-1` means unlimited
+1. `ships` - The list of available ships on this station
+	1. `shipyard_list` - The actual list (Dictionary, ship name as key)
+		1. `id` - ShipId
+		1. `name` - Ship name
+		1. `basevalue` - The base cost of a new ship
+		1. `sku` - The SKU for the ship
+		1. `stock` - How many of this ship we have in stock
+
+&nbsp;
```

Comment:
A non-breaking space, why ?

Reply:
Was just to get extra spacing below the list.

Reply:
Eh, there's a horizontal rule before the next section anyway.